.gitignore

```
@@ -38,3 +38,5 @@
 /config/database.yml
 
 /Gemfile.lock
+
+/.idea/todo_list.iml
```

In case there are more files that are generated by the IDE
```suggestion
/.idea/*
```